### DISCUSSION TITLE
How to get maps from 2 different vacuums?

### DISCUSSION THREAD

**Discussion by lynxchat4** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#387] [Q&A] [answered]
Hello!
I have 2 Roborock Vacuum Cleaners.  
a15 (s7 plus)
a46 (g10s)
How to get maps from 2 different vacuums?

**Comment by PiotrMachowski** [comment, marked as answer]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/discussions/108
